v1/formats/Spanish.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<debate-format schema-version="2.2">
+  <name xml:lang="en">Spanish Academic</name>
+  <name xml:lang="es">Académico español</name>
+  <version>2</version>
+  <languages>
+    <language>en</language>
+    <language>es</language>
```

Comment:
Do you want to list Spanish before English, since it's a Spanish format?

(This only matters in phones that list neither Spanish nor English in their language settings. If the user uses neither language or anything close to it, it should fall back to whatever's listed first here.)